model/riscv_sys_regs.sail

```diff
@@ -826,6 +881,15 @@ bitfield Vtype  : xlenbits = {
 }
 register vtype : Vtype
 
+//Returns the PMM bits value depends on the Privilege mode
+function get_pmm() -> bits(2) = {
```

Comment:
Oh wait... you already had that here. Even simpler then!

```
function is_pmm_active() -> bool = get_pmm() != zeros()
```

Probably can just inline that actually.